tests/core/test_data.py

```diff
@@ -105,17 +106,14 @@ def test_same_file_names_get_resolved(tmpdir):
     assert all([f.endswith("stories.md") for f in stories])
 
 
-@pytest.mark.parametrize(
-    "line",
-    [
-        "##intent:aintent",
-        "##synonym: synonym",
-        "##regex:a_regex",
-        " ##lookup:additional",
-    ],
-)
-def test_contains_nlu_pattern(line):
-    assert data._contains_nlu_pattern(line)
+def test_find_core_nlu_files_in_directory():
```

Comment:
Can you have the test case reflect what it's doing? E.g. `test_finding_of_nlu_files_with_multiple_formats`

Reply:
https://docs.pytest.org/en/latest/parametrize.html